### PR TITLE
Clean up justified/unjustified layouts. Fix some more whitespace.

### PR DESCRIPTION
**Pull request by d7415**

### CODE REVIEW

**Comment by d7415** [comment]
See also LycheeOrg/Lychee#212

**Comment by d7415** [comment]
(Also, please either straight merge or wait for me as I've used the refs in the Lychee PR)

**Comment by d7415** [comment]
(Laravel awaiting me figuring it out :P)

**Comment by d7415** [comment]
> value needs to be "2"

Good catch. I told @ildyria I'd create bugs!

File: scripts/view.js [hunk 1]
```diff
@@ -205,10 +205,10 @@ view.album = {
 			}
 
 			if (photosData !== '') {
-				if (lychee.justified) {
+				if (lychee.layout == '1') {
 					photosData = '<div class="justified-layout">' + photosData + '</div>';
 				}
-				else if (lychee.unjustified) {
+				else if (lychee.layout == '2') {
```
Comment (ildyria) [inline scripts/view.js]: are you sure it should not be `===` instead of just `==` ?

File: scripts/view.js [hunk 2]
```diff
@@ -231,7 +231,7 @@ view.album = {
 			// Add photos to view
 			lychee.content.html(html);
 			view.album.content.justify();
-			if (lychee.unjustified) {
+			if (lychee.layout == '2') {
```
Comment (ildyria) [inline scripts/view.js]: are you sure it should not be `===` instead of just `==` ?

File: scripts/view.js [hunk 3]
```diff
@@ -288,7 +288,7 @@ view.album = {
 		},
 
 		justify: function () {
-			if (!lychee.justified) return;
+			if (lychee.layout != '1') return;
```
Comment (ildyria) [inline scripts/view.js]: are you sure it should not be `!==` instead of just `!=` ?

**Comment by d7415** [comment]
I think you're right. I'll change it (and the dist/submodules) when I have some time later.

Thanks!

**Pull request merged**